package.json

```diff
@@ -1,26 +1,24 @@
 {
-  "name": "asciidoctor-reveal.js",
+  "name": "@asciidoctor/reveal.js-converter",
   "version": "2.0.1-dev",
-  "description": "A Reveal.js converter for Asciidoctor and Asciidoctor.js. Write your slides in AsciiDoc!",
+  "description": "A Reveal.js converter for Asciidoctor.js. Write your slides in AsciiDoc!",
   "main": "dist/main.js",
   "engines": {
-    "node": ">=4",
-    "npm": ">=3.0.0"
+    "node": ">=8.11",
+    "npm": ">=5.0.0"
   },
-  "files": [
-    "dist",
-    "LICENSE.adoc",
-    "README.adoc"
-  ],
+  "files": [],
```

Comment:
why change this?

Reply:
By default "LICENSE*", "README*" and the target of `main` are included: https://docs.npmjs.com/files/package.json#files

I can move this change to another pull request, if you want.

Reply:
No, it's good.